setup.py

```diff
@@ -99,6 +99,7 @@
     "sklearn.metrics._pairwise_distances_reduction._middle_term_computer",
     "sklearn.metrics._pairwise_distances_reduction._base",
     "sklearn.metrics._pairwise_distances_reduction._argkmin",
+    "sklearn.metrics._pairwise_distances_reduction._argkminclassmode",
```

Comment:
For consistency, I think the introduced extension better be named `_argkmin_classmode` instead of `_argkminclassmode`. What do you think?
```suggestion
    "sklearn.metrics._pairwise_distances_reduction._argkmin_classmode",
```